src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -658,11 +675,12 @@ private boolean checkForPickaxe(final int minlevel)
             {
                 return true;
             }
-            requestWithoutSpam("Pickaxe at least level " + minlevel);
+            requestWithoutSpam("Pickaxe of at least level " + minlevel + " and at maximum " + hutLevel);
```

Comment:
Can we put this in our languageHandler and lang files so we're able to translate it?